Modules/Test/classes/class.ilTestRandomQuestionSetConfig.php

```diff
@@ -384,7 +384,7 @@ public function removeQuestionSetRelatedData(): void
 
     public function resetQuestionSetRelatedTestSettings()
     {
-        $this->testOBJ->setResultFilterTaxIds(array());
```

Comment:
need some expertise with this

Reply:
Hi @mbecker-databay, 

could you please give @fhelfer a hand?

Reply:
I think I fixed this particular one yesterday... 

        $this->testOBJ->getScoreSettingsRepository()->store(
            $this->testOBJ->getScoreSettings()->withResultDetailsSettings(
                $this->testOBJ->getScoreSettings()->getResultDetailsSettings()->withTaxonomyFilterIds(array())
            )
        );